talisker/gunicorn.py

```diff
@@ -219,7 +219,7 @@ def load_config(self):
                 extra={'errorlog': self.cfg.errorlog})
             self.cfg.set('errorlog', '-')
 
-        if self.cfg.loglevel.lower() == 'debug':
+        if self.cfg.loglevel.lower() == 'debug' and self._devel:
```

Comment:
Yes, we don't ever want to send DEBUG logs to stderr in prod, as they are a) sensitive and b) voluminous, so we force devel mode